recipes/gf2x/all/conanfile.py

```diff
@@ -0,0 +1,157 @@
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def build_requirements(self):
+        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
+
+        if self._settings_build.os == "Windows":
+            self.build_requires("strawberryperl/5.30.0.1")
+
+        self.build_requires("m4/1.4.19")
+        self.build_requires("autoconf/2.71")
+            
+        if is_msvc(self):
+            self.build_requires("yasm/1.3.0")
+            self.build_requires("automake/1.16.4")
+
+    def _configure_autotools(self):
+        with chdir(self, self._source_subfolder):
+            self.run(
+                "{} --install".format(os.environ["AUTORECONF"]),
+                win_bash=tools.os_info.is_windows,
+                run_environment=True
+            )
+
+        yes_no = lambda v: "yes" if v else "no"
+        autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        configure_args = [
+            "--enable-hardware-specific-code=%s" % yes_no(self.options.hardware_specific_code),
+            "--enable-sse2=%s" % yes_no(self.options.sse2),
+            "--enable-sse3=%s" % yes_no(self.options.sse3),
+            "--enable-ssse3=%s" % yes_no(self.options.ssse3),
+            "--enable-sse41=%s" % yes_no(self.options.sse41),
+            "--enable-pclmul=%s" % yes_no(self.options.pclmul),
+            "--enable-fft-interface=%s" % yes_no(self.options.fft_interface)
+        ]
+        autotools.configure(args=configure_args, configure_dir=self._source_subfolder)
+
+        return autotools
+
+    @contextlib.contextmanager
+    def _build_context(self):
+        if is_msvc(self):
+            with tools.vcvars(self):
+                yasm_machine = {
+                    "x86": "x86",
+                    "x86_64": "amd64",
+                }[str(self.settings.arch)]
+                env = {
+                    "CC": "cl -nologo",
+                    "CCAS": "{} -a x86 -m {} -p gas -r raw -f win32 -g null -X gnu".format(os.path.join(self.build_folder, "yasm_wrapper.sh").replace("\\", "/"), yasm_machine),
+                    "CXX": "cl -nologo",
+                    "AR": "{} lib".format(os.environ["AUTOMAKE"]),
+                    "LD": "link -nologo",
+                    "NM": "python {}".format(tools.unix_path(os.path.join(self.build_folder, "dumpbin_nm.py"))),
+                }
+
+                with tools.environment_append(env):
+                    yield
+        else:
+            yield
+
+    def build(self):
+        with self._build_context():
+            autotools = self._configure_autotools()
+            autotools.make()
+
+    def package(self):
+        copy(self, "COPYING", src=self._source_subfolder, dst=os.path.join(self.package_folder, "licenses"))
```

Comment:
```suggestion
        copy(self, "COPYING*", src=self._source_subfolder, dst=os.path.join(self.package_folder, "licenses"))
```
Otherwise, it will copy only the GPL-3.0 file

Reply:
Done